### PR TITLE
NLP-ENGINE-030. Remove global VTRun_Ptr. [DEGLOB]

### PR DESCRIPTION
**Pull request by amnonma**
Signed-off-by: amnonma <amnon@textanalysis.com>

### CODE REVIEW

File: include/Api/lite/nlp.h [hunk 1]
```diff
@@ -86,7 +87,8 @@ class LITE_API NLP
 		_TCHAR *outdir = 0,     // DIRECTORY FOR OUTPUT.			  // 03/10/99 AM.
 //		char *datadir = 0			// VISUALTEXT DATA DIRECTORY		  // 12/08/99 AM.
       _TCHAR *name = 0,       // UNIQUE NAME FOR ANALYZER.       // 01/06/03 AM.
-      _TCHAR *logfile = 0     // VTLOG	// 05/06/13 AM.
+      _TCHAR *logfile = 0,     // VTLOG	// 05/06/13 AM.
+	  VTRun *vtrun = 0			// VT Runtime Environment.	// [DEGLOB]	// 10/15/20 AM.
```
Comment (dehilsterlexis) [inline include/Api/lite/nlp.h:91]: Dummy comment here.

**Review by dehilsterlexis — CHANGES_REQUESTED** [review]
Yea, make the change

**Comment by amnonma** [comment]
@dehilsterlexis Please review.

**Comment by dehilsterlexis** [comment]
@amnonma look good

**Pull request merged**